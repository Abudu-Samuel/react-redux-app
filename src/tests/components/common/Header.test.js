/* eslint-disable */
import Header from '../../../components/common/Header';

describe('<Header />', () => {
  it('render Header component', () => {
    const wrap = shallow(<Header />);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});