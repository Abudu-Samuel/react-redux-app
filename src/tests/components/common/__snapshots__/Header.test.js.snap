// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> render Header component 1`] = `
<nav>
  <Link
    className="active"
    replace={false}
    to="/"
  >
    Home
  </Link>
   | 
  <Link
    className="active"
    replace={false}
    to="/courses"
  >
    Courses
  </Link>
   | 
  <Link
    className="active"
    replace={false}
    to="/about"
  >
    About
  </Link>
</nav>
`;
